image_width: 640
image_height: 480
camera_name: camera
camera_matrix:
  rows: 3
  cols: 3
  data: [835.387925, 0.000000, 256.516012, 0.000000, 835.383355, 248.860166, 0.000000, 0.000000, 1.000000]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.049828, -0.138885, 0.009937, -0.015179, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
projection_matrix:
  rows: 3
  cols: 4
  data: [836.642090, 0.000000, 250.120799, 0.000000, 0.000000, 846.013489, 251.593089, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]